staging/cse/windows/provisioningscripts/kubeletstart.ps1

```diff
@@ -66,6 +76,22 @@ if ($global:ContainerRuntime -eq "containerd") {
     $KubeletArgList += @("--container-runtime=remote", "--container-runtime-endpoint=npipe://./pipe/containerd-containerd")
 }
 
+# Update args to get both IPs from NIC using the default route 0.0.0.0/0 if IsDualStackEnabled
+# and the --node-ip argument is not present in the existing kubelet arg list.
+if ($global:IsDualStackEnabled -and !(Test-NodeIPArgExists $KubeletArgList)) {
```

Comment:
This script will be run by nssm.exe as a service. How will `IsDualStackEnabled` be set?

Reply:
ah that's good to know. I updated it to reference the kubeproxy feature gates which will have "IPv6DualStack=true" instead